app/views/nic/_base_form.html.erb

```diff
@@ -17,16 +17,18 @@
 <%= text_f f, :name, :label => _("DNS name"), :value => f.object.shortname,
               :help_inline => _("Primary interface's DNS name and domain define host's FQDN"),
               :class => :interface_name %>
-<%= select_f f, :domain_id, accessible_domains, :id, :to_label,
-             { :include_blank => accessible_domains.any? ? true : _("No domains")},
-             { :disabled => accessible_domains.empty? ? true : false,
-               :help_inline => :indicator,
-               :class => :interface_domain, :'data-url' => domain_selected_hosts_path } %>
-<%= select_f f, :subnet_id, accessible_subnets, :id, :to_label,
-             { :include_blank => accessible_subnets.any? ? true : _("No subnets")},
-             { :disabled => accessible_subnets.empty? ? true : false,
-               :help_inline => :indicator,
-               :class => :interface_subnet, :'data-url' => freeip_subnets_path } %>
+<% if SETTINGS[:unattended] %>
+  <%= select_f f, :domain_id, accessible_domains, :id, :to_label,
```

Comment:
I kind of agree, the only trouble is that the host or NIC models still seem to assign a domain based on the FQDN I enter if I have a matching one in the database.  It might be useful to show it, if only so it can be unset to fix a mess?